### PR TITLE
BaSD-MS-81: Create Shared modal

### PR DESCRIPTION
**Pull request by Micaela-89**
Create a SHARED MODAL.

 Refactor in all components that are using MODALS.

### CODE REVIEW

File: src/Components/Psychologists/Modal/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import styles from './modal.module.css';
 
-const Modal = ({
```
Comment (veronabaudracco) [inline src/Components/Psychologists/Modal/index.js:4]: You should rename this modal to "AvailabilityModal"

**Review by veronabaudracco — CHANGES_REQUESTED** [review]

File: src/Components/Postulants/postulants.module.css [hunk 2]
```diff
@@ -46,7 +46,7 @@
   text-decoration: none;
   font-weight: bold;
   text-decoration: none;
-  padding: 5px;
+  padding: 5px; */
```
Comment (fjmorabito) [inline src/Components/Postulants/postulants.module.css]: Delete comments

**Review by fjmorabito — CHANGES_REQUESTED** [review]
Delete comments on postulants.module.css

**Review by fjmorabito — APPROVED** [review]

**Review by veronabaudracco — APPROVED** [review]
Good job!

**Pull request merged**